### PR TITLE
November Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | ea5799d |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-552.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/2024nov_amplify.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    The Agent SDK v1.1.102 has been updated so all agents default to use GRPC mode instead of polled mode (CENTRAL_GRPC_ENABLED=true).  This will improve the efficiency of the communication between the agents and the platform
+
+<!--### Agents known issues (hidden if none)-->
+
+* **UI for Agent Management**
+
+    ENHANCEMENT<br />
+    The Agent Management screen has been moved to a subpage under Topology -> Environments.  The first column displays the Agent Title and logical name and the Dataplane column has been renamed to Environment to improve the User Experience.
+
+* **UI for Service Regostry**
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/2024nov_amplify.md]: **Jump to a specific page in Service Registry**

ENHANCEMENT<br />
Users can now **jump directly to a specific page** in the Service Registry, using the pagination control that is placed at the buttom of the table. Simply click on the "..." in the pagination controls to quickly access the page selector popup, where you can select the page you want to navigate to. This removes the need to having to click through multiple pages, saving time when browsing large lists of services.

File: content/en/docs/amplify_relnotes/2024nov_amplify.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+The latest version of the [Axway Central CLI is v3.7.0 on NPM](https://www.npmjs.com/package/@axway/axway-central-cli/v/3.7.0). For details on Central CLI commands, see [Axway Central CLI Command reference](/docs/integrate_with_central/cli_central/cli_command_reference).
+
+### Axway CLI updates for November ?
+
+The latest version of the [Axway CLI is v3.2.14 on NPM](https://www.npmjs.com/package/@axway/axway/v/3.2.14).
+
+### Marketplace updates for November 1st
+
+* **Custom unit quota**
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/2024nov_amplify.md:64]: **Custom consumption units for billing and usage tracking**

PROVIDER BILLING; NEW FEATURE<br />
We are thrilled to introduce a powerful new feature that gives users the flexibility to define custom consumption units for more accurate and adaptable usage-based charging. This enhancement is designed to meet the evolving needs of service providers, particularly with the rise of generative AI and Large Language Model (LLM) services.
 * **Custom Consumption Unit**: Users can now define custom units for measuring and charging API usage. Previously, usage metrics were limited to charging by transactions only. This feature is especially valuable for providers offering generative AI services, where usage metrics like tokens are more appropriate.
 * **Use custom unit in plan quota**: Custom consumption units can now be used in the **product plan quota** definitions.

Refer to [Manage consumption unit](/docs/manage_product_foundry/manage_consumption_units) for more details.

File: content/en/docs/amplify_relnotes/2024nov_amplify.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+   PROVIDER BILLING<br />
+   Currently only Transactions unit are monitored. But with the rising of LLM model, new units are necessary to charge consumer. By default the system manages only **Transaction** unit. When defining your plan quota, you can select any other consumption unit available in the system. Refer to [Manage consumption unit](/docs/manage_product_foundry/manage_consumption_units).
+
+* **Async API 3.0.0 support**
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/2024nov_amplify.md:69]: **Async API 3.0.0 support**

CONSUMER EXPERIENCE, ENHANCEMENT<br />
The component responsible for rendering API specifications has been upgraded to support **AsyncAPI v3.0.0**.
Users can now enjoy a visually appealing and well-structured presentation of AsyncAPI specifications directly in the Marketplace. In addition, when importing an AsyncAPI 3.0.0 specification into the **Service Registry**, the system will now automatically validate the definition. This ensures that only properly defined and compliant specifications are registered.

File: content/en/docs/amplify_relnotes/2024nov_amplify.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+   CONSUMER EXPERIENCE, ENHANCEMENT<br />
+   The component managing the display of API specification has been upgrades to display the Async API v3.0.0. The component validate the specification definition while importing the specification in Service Registry and allow a nice visual on the Marketplace.
+
+* **Enhancement title**
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/2024nov_amplify.md:74]: @cbordier , probably we need to remove this section before we publish. This is just part of the template.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

File: content/en/docs/amplify_relnotes/2024nov_amplify.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+* **Jump to a specific page in Service Registry**
+
+    ENHANCEMENT<br />
+    When multiple pages are available for the **Service Registry** to display, simply click the "..." in the pagination controls that is place at the button of the page and select the page you want to navigate to. This makes it easier to browse large lists of Services in the Service Registry.
+
+### Agents bug fixes for November 1st
+
+| Case ID     | Internal ID  | Description                                       |
+|-------------|--------------|---------------------------------------------------|
+| 01647785<br /> 01650304 | APIGOV-29069 | **Issue**: Axway API Management agent did not reflect the transision from "unhealthy" to a "running" states on the Ui. <br />**Resolution**: A fix was made to the agent state correctly. |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/2024nov_amplify.md:53]: *transition*...*state*...*UI*

"...made to *represent* the agent state..."

File: content/en/docs/amplify_relnotes/2024nov_amplify.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+|             | APIGOV-29048 | **Issue**: Auto-release of a new asset version was not being triggered by CRD or ARD changes. <br />**Resolution**: A fix was made to the auto-release an asset version when ARD or CRD changes are made. |
+| 01640474    | APIGOV-29157 | **Issue**: Axway API Management agent did not report metrics for a specific API. <br />**Resolution**: A fix was made to the handling of application name to check that the base path matches the path reported in the event. |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/2024nov_amplify.md:55]: "...for a specific API when that API name is duplicated amongst services."

File: content/en/docs/amplify_relnotes/2024nov_amplify.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+|             | APIGOV-29142 | **Issue**: Axway API Management Discovery agent would not be able to get the information needed to publish a proxy. <br />**Resolution**: A fix was made add retry logic to make additional attempts to get the information needed to publish a proxy. |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/2024nov_amplify.md:56]: "A fix was made *to* add retry..."

File: content/en/docs/amplify_relnotes/_index.md [hunk 8]
```diff
@@ -7,27 +7,52 @@ date: 2020-10-28
...
 
 To view the *Administration Release Notes* [click here](https://docs.axway.com/bundle/platform-management/page/docs/release_notes/index.html).
 
 To view the *Support Policy for Amplify Agents and Amplify Agent SDK* [click here](/docs/amplify_relnotes/agent_agentsdk_support_policy/).
 
+## [Amplify Enterprise Marketplace November ? 2024](/docs/amplify_relnotes/2024nov_amplify/#november-?-2024)
+
+| Environment type                        | Latest on-premise agent version <br />(based on Amplify Agents SDK v1.1.102 <br />unless otherwise noted) | Minimum supported agent version <br />(based on Amplify Agents SDK v1.1.83) |
+|--------------------------------------------|--------------------------|------------------|
+| Axway API Management 7.6.2 SPx and 7.7 SPx | DA=v1.2.9 / TA=v1.2.7    | DA=v1.1.80 / TA=v1.1.81          |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/_index.md:24]: Are we using a semantic version with or with out a "v" prefix? It is very mixed here. Should either be vX.Y.Z or just X.Y.Z, not mixed.

File: content/en/docs/amplify_relnotes/_index.md [hunk 9]
```diff
@@ -7,27 +7,52 @@ date: 2020-10-28
...
 
 {{< alert title="Note" color="primary" >}}**Unified Catalog Deprecation Notice** - As of August 8, 2024, the Unified Catalog is no longer supported.{{< /alert >}}
 
 ### October 18 Supported agents
 
-Update
-
 | Environment type                        | Latest on-premise agent version <br />(based on Amplify Agents SDK v1.1.101 <br />unless otherwise noted) | Minimum supported agent version <br />(based on Amplify Agents SDK v1.1.83) |
 |--------------------------------------------|--------------------------|------------------|
 | Axway API Management 7.6.2 SPx and 7.7 SPx | DA=v1.2.7 / TA=v1.2.6    | DA=v1.1.80 / TA=v1.1.81          |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/_index.md:58]: Why is this version not updated?

File: content/en/docs/amplify_relnotes/2024nov_amplify.md (hunk 5 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/2024nov_amplify.md:53]: edited

File: content/en/docs/amplify_relnotes/2024nov_amplify.md (hunk 7 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/2024nov_amplify.md:56]: edited

File: content/en/docs/amplify_relnotes/_index.md (hunk 8 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/_index.md]: i removed the V

File: content/en/docs/amplify_relnotes/_index.md (hunk 9 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/_index.md:58]: Look at the heading , those are the correct version numbers for "October 18 Supported Agents"

**Pull request merged**